packages/opc-back-to-top/src/opc-back-to-top.scss

```diff
@@ -0,0 +1,21 @@
+@import '@one-platform/opc-styles';
+
+$LOCAL: opc-back-to-top;
+
+$variables: ( 
+  //Add your variables here
+);
+
+.hide{
```

Comment:
can u provide a space between the .hide class and opening curly brace i.e `.hide {`